<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviewDialog</class>
 <widget class="QDialog" name="PreviewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>439</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>571</width>
    <height>439</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>571</width>
    <height>439</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PreviewDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="1" column="0">
    <widget class="QPushButton" name="prevBtn">
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="closeBtn">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QStackedWidget" name="prevWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="imagePage">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="imgPrev">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="AudioPage">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="stopBttn">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pauseBttn">
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="playBttn">
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="AudioSeekBar" name="progressBar">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
         <property name="tickInterval">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="progDisplay">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="FontPage">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer malesuada laoreet leo. Quisque in arcu a quam porta tincidunt in at lacus. Sed sed ipsum eu sapien laoreet ullamcorper. Aenean vel lacinia arcu. Mauris quis lectus vitae purus aliquam volutpat. Sed bibendum nibh vel dictum tempor. In hac habitasse platea dictumst. Integer vitae viverra quam. Proin eu diam condimentum, sollicitudin nisl ut, porttitor dolor. Fusce tortor est, eleifend sed tincidunt ac, pulvinar quis est. Morbi adipiscing facilisis justo, eget interdum odio volutpat in. Nunc enim mauris, molestie at mauris et, cursus condimentum tortor. Quisque viverra ultrices turpis eget cursus. Aenean sed vehicula massa, ac euismod augue.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="UnknownPage">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>No preview available</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="nextBtn">
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AudioSeekBar</class>
   <extends>QSlider</extends>
   <header>AudioSeekBar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
